hcaptcha_field/fields.py

```diff
@@ -108,6 +109,17 @@ def widget_attrs(self, widget):
             attrs['data-%s' % key] = value
         return attrs
 
+
+    def _get_ssl_context(self):
+        tls = None
+        if hasattr(ssl, 'PROTOCOL_TLSv1'):
+            tls = ssl.PROTOCOL_TLSv1
+        if hasattr(ssl, 'PROTOCOL_TLS'):
+            tls = ssl.PROTOCOL_TLS
+        if hasattr(ssl, 'PROTOCOL_TLSv1_2'):
+            tls = ssl.PROTOCOL_TLSv1_2
```

Comment:
All these constants have been deprecated since version 3.6 and 3.10 ([docs](https://docs.python.org/3/library/ssl.html#constants)), so I do not think this is the desired solution.